temoa/data_processing/database_util.py

```
@@ -41,361 +47,284 @@ def __init__(self, database_path, scenario=None):
-            raise ValueError('For Output related queries, please set a scenario first')
-        columns = []
+    def get_output_flow_for_period(
+        self,
+        period: int,
+        region: str | None,
+        comm_type: str = 'input',
+        commodity: str | None = None,
+    ) -> pd.DataFrame:
+        """Retrieves aggregated output flow data."""
+        if not self.scenario:
+            raise ValueError('A scenario must be set for output-related queries')
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)